src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/producer/impl/MessageProducerImpl.java

```diff
@@ -39,7 +39,6 @@ public void onSuccess(SendResult<String, ChipsRestInterfacesSend> result) {
                 Map<String, Object> logMap = new HashMap<>();
                 logMap.put(KEY_OFFSET, result.getRecordMetadata().offset());
                 logMap.put(KEY_PARTITION, result.getRecordMetadata().partition());
-                logMap.put(KEY_MESSAGE, chipsMessage.getData());
 
                 logger.infoContext(messageId, "Sent messageId " + messageId + " to topic " + topicName, logMap);
             }
```

Comment:
I can't put this comment on the actual line but on line 48, might be worth removing chipsMessage from the exception. It's little hidden ones like this that get you!